src/proc.cr

```diff
@@ -62,7 +62,7 @@
 # ```
 # module Ticker
 #   # The callback for the user doesn't have a Void*
-#   @@box : Box(Int32 ->)
+#   @@box : Pointer(Void)?
```

Comment:
@Sija 
`Box(T).box` returns `Void*`. And, it seems this minimum code works with only `Pointer(Void)`.
```crystal
box : Pointer(Void) = Box.box(1) # => Pointer(Void)@0x267ffc0
box : Box(Int32)    = Box.box(1) # type must be Box(Int32), not Pointer(Void)
```